app/api/src/players/players.controller.ts

```diff
@@ -3,39 +3,55 @@ import { ApiOperation } from '@nestjs/swagger';
 
 import { PlayersService } from './players.service';
 import { PaginationQueryDto } from '../common/dto/pagination-query.dto';
-import { TopPlayersQueryDto } from '../common/dto/top-players-query.dto';
+import {
+	TopAccuracyQueryDto,
+	TopWinsQueryDto,
+} from '../common/dto/top-players-query.dto';
 
 @Controller('players')
 export class PlayersController {
 	constructor(private readonly playerService: PlayersService) {}
 
 	// /players
 	@Get()
-	@ApiOperation({ tags: [ 'Player' ], summary: 'Return a list of players' })
+	@ApiOperation({ tags: ['Player'], summary: 'Return a list of players' })
```

Comment:
Some of these are just formatting changes from applying Prettier; it doesn't support bracket spacing for `[ ]`, only `{ }`